lang/lowering/src/lower/exp/binop.rs

```diff
@@ -5,22 +5,34 @@ use crate::{Ctx, LoweringResult, lower::Lower};
 impl Lower for cst::exp::BinOp {
     type Target = ast::Exp;
     fn lower(&self, ctx: &mut Ctx) -> LoweringResult<Self::Target> {
-        let cst::exp::BinOp { span, operator, lhs, rhs } = self;
+        let cst::exp::BinOp { span, lhs, rhs } = self;
```

Comment:
We should probably add a comment that all operators are currently left-associative.

Reply:
Right-associative, no? `a + b + c`gets parsed as `a + (b + c)`.